src/nyc_trees/apps/event/tests.py

```diff
@@ -6,6 +6,7 @@
 from functools import partial
 
 from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
```

Comment:
I used `timedelta` on other event code, should we be using `relativedelta` throughout, instead?